tanner/tests/test_sqlite.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+import os
+import sqlite3
+import unittest
+from unittest import mock
+
+from tanner.emulators import sqlite
+
+class SqliteTest(unittest.TestCase):
+	def setUp(self):
+		self.loop = asyncio.new_event_loop()
+		asyncio.set_event_loop(None)
+		self.filename = '/tmp/db/test_db'
+		os.makedirs(os.path.dirname(self.filename), exist_ok=True)
+		open('/tmp/db/test_db', 'a').close()
+		# Insert some testing data
+		conn = sqlite3.connect(self.filename)
+		self.cursor = conn.cursor()
```

Comment:
Maybe we shouldn't use real db interactions in the tests? We just want to test our logic, not db interactions itself. How do you think?